app/models/katello/glue/elastic_search/system.rb

```diff
@@ -1,153 +0,0 @@
-# Comments below are meant to describe in detail how to use elasticsearch (ES) properly. Systems
-# are most commonly used as a reference for adding other model/view/controllers to katello.
-
-# rubocop:disable SymbolName
-module Katello
-  module Glue::ElasticSearch::System
-    # rubocop:disable MethodLength
-    def self.included(base)
-      base.class_eval do
-        include Ext::IndexedModel
-
-        add_host_collection_hook lambda { |host_collection| reindex_on_association_change(host_collection) }
```

Comment:
You should see these hooks and their associated functions in other places such as the system model itself -- they can be removed with this change.